tutorials/examples/preprocess_imgs.py

```diff
@@ -0,0 +1,123 @@
+"""
+This script illustrates the preprocess steps before starting the pipeline of
+reconstructing 3D map using the simulated 2D images.
+"""
+import logging
+import matplotlib.pyplot as plt
+import mrcfile
+import numpy as np
+import os
+
+from aspire.estimation.noise import WhiteNoiseEstimator
+from aspire.source.simulation import Simulation
+from aspire.utils.filters import (RadialCTFFilter, ScalarFilter)
+from aspire.utils.preprocess import downsample
+
+
+logger = logging.getLogger('aspire')
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), '../data/')
+
+logger.info('This script illustrates orientation estimation using '
+            'synchronization matrix and voting method')
+
+# Set the downsample size of images
+img_size = 33
+
+# Set the total number of images generated from the 3D map
+num_imgs = 512
+
+# Set the number of 3D maps
+num_maps = 1
+
+# Set the noise variance and build the noise filter
+noise_variance = 4e-1
+noise_filter = ScalarFilter(dim=2, value=noise_variance)
+
+# Specify the CTF parameters not used for this example
+# but necessary for initializing the simulation object
+pixel_size = 5*65/img_size       # Pixel size of the images (in angstroms).
```

Comment:
Spaces around binary operators. No period at the end of comment.

Reply:
Done.